src/renderers/shaders/ShaderChunk/envmap_common_pars_fragment.glsl.js

```diff
@@ -1,6 +1,7 @@
 export default /* glsl */`
 #ifdef USE_ENVMAP
 
+	uniform float envMapBlurriness;
```

Comment:
We are not blurring the environment map. We are blurring the background.

Reply:
Updated!